### PR TITLE
People Update MD

### PR DESCRIPTION
**Pull request by bfossen-ce**
People Update MD

### CODE REVIEW

**Comment by sroomf** [comment]
lots of lint errors related to redistricting for this one

File: data/md/legislature/Jessica-Feldmark-03575039-449a-4378-a021-ebdd11a107f4.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+id: ocd-person/03575039-449a-4378-a021-ebdd11a107f4
+name: Jessica Feldmark
+given_name: Jessica
+family_name: Feldmark
+image: http://mgaleg.maryland.gov/2023RS/images/feldmark01.jpg
+party:
+- name: Democratic
```
Comment (maxknutsen) [inline data/md/legislature/Jessica-Feldmark-03575039-449a-4378-a021-ebdd11a107f4.yml:7]: @sroomf this person should have a file already and just needs added:
- type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:md/government
  district: 12A
They previously served in house district 12 from jan 2019 until 01/11/2023

File: data/md/legislature/Jon-S-Cardin-48959ab9-e2d5-4928-840d-bd0ca5d86696.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+id: ocd-person/48959ab9-e2d5-4928-840d-bd0ca5d86696
+name: Jon S. Cardin
+image: http://mgaleg.maryland.gov/2023RS/images/cardin01.jpg
+party:
+- name: Democratic
+roles:
```
Comment (maxknutsen) [inline data/md/legislature/Jon-S-Cardin-48959ab9-e2d5-4928-840d-bd0ca5d86696.yml:6]: @sroomf this person should already have a file elsewhere and we just need to add:
- type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:md/government
  district: 11B
They previously were in house district 11 from Jan 2019 until 01/11/2023

File: data/md/legislature/Mike-Griffith-c3b6f47e-90e1-4abf-af49-c5c6fd7993a2.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+id: ocd-person/c3b6f47e-90e1-4abf-af49-c5c6fd7993a2
+name: Mike Griffith
+given_name: Mike
+family_name: Griffith
+email: mike.griffith@house.state.md.us
+image: http://mgaleg.maryland.gov/2023RS/images/griffith02.jpg
+party:
+- name: Republican
```
Comment (maxknutsen) [inline data/md/legislature/Mike-Griffith-c3b6f47e-90e1-4abf-af49-c5c6fd7993a2.yml:8]: @sroomf this person should already have a file, we just need to add:
- type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:md/government
  district: 35A
They previously served in house district 35b from 01/07/2020-01/11/2023

File: data/md/retired/Mike-McKay-5bdf592a-c502-4b60-8799-8ed681950475.yml [hunk 4]
```diff
@@ -8,14 +8,10 @@ image: http://mgaleg.maryland.gov/2022RS/images/mckay01.jpg
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2023-01-11'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:md/government
   district: 1C
```
Comment (maxknutsen) [inline data/md/retired/Mike-McKay-5bdf592a-c502-4b60-8799-8ed681950475.yml:14]: @sroomf move to leg.

File: data/md/legislature/Nino-Mangione-346c3339-2f88-4d95-9888-546174d081fb.yml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+id: ocd-person/346c3339-2f88-4d95-9888-546174d081fb
+name: Nino Mangione
+given_name: Nino
+family_name: Mangione
+image: http://mgaleg.maryland.gov/2023RS/images/mangione01.jpg
+party:
+- name: Republican
```
Comment (maxknutsen) [inline data/md/legislature/Nino-Mangione-346c3339-2f88-4d95-9888-546174d081fb.yml:7]: @sroomf this person should have a file already and just needs added:
- type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:md/government
  district: 42A
they previously were in house 42b from jan 2019 until 01/11/2023.

File: data/md/retired/Ken-Kerr-38379d8c-a63d-4e2b-88ee-ee6dcd646c95.yml [hunk 6]
```diff
@@ -4,18 +4,18 @@ given_name: Ken
...
-image: http://mgaleg.maryland.gov/2022RS/images/kerr01.jpg
+image: http://mgaleg.maryland.gov/2023RS/images/kerr01.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2023-01-11'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:md/government
   district: 3B
```
Comment (maxknutsen) [inline data/md/retired/Ken-Kerr-38379d8c-a63d-4e2b-88ee-ee6dcd646c95.yml:14]: @sroomf move to leg.

File: data/md/retired/Terri-L-Hill-9b63f31e-5bc0-47bf-a901-0a8dcc2a67b7.yml [hunk 7]
```diff
@@ -4,17 +4,18 @@ given_name: Terri
...
-image: http://mgaleg.maryland.gov/2022RS/images/hill02.jpg
+image: http://mgaleg.maryland.gov/2023RS/images/hill02.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2023-01-11'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:md/government
   district: '12'
```
Comment (maxknutsen) [inline data/md/retired/Terri-L-Hill-9b63f31e-5bc0-47bf-a901-0a8dcc2a67b7.yml:14]: @sroomf move to leg.

**Review by maxknutsen — COMMENTED** [review]
@sroomf a couple comments but looks good! This one was easy :)

**Review by maxknutsen — APPROVED** [review]
all good!

**Pull request merged**